src/at_commands/at_ast_definition.rs

```diff
@@ -36,8 +36,7 @@ pub async fn results2message(result: &AstQuerySearchResult) -> Vec<ContextFile>
     symbols
 }
 
-async fn run_at_definition(ast: &Option<Arc<ARwLock<AstModule>>>, symbol: &String) -> Result<Vec<ContextFile>, String>
-{
```

Comment:
absolutely unneded changes
remove this refactoring here and below

Reply:
I seriously dislike this code formatting